tests/test_comparison_level_composition.py

```diff
@@ -72,16 +52,47 @@ def test_binary_composition_internals(clause, c_fun):
         m_probability=0.5,
     )
 
-    exact_match_sql = (
-        f'("first_name_l" = "first_name_r") {clause} ("surname_l" = "surname_r")'
-    )
+    exact_match_sql = f"({q}first_name_l{q} = {q}first_name_r{q}) {clause} ({q}surname_l{q} = {q}surname_r{q})"  # noqa: E501
     assert level.sql_condition == f"NOT ({exact_match_sql})"
 
     with pytest.raises(ValueError):
         c_fun()
 
 
-def test_composition_outputs():
+@mark_with_dialects_excluding("postgres")
```

Comment:
Running these locally it seems like they all support postgres without modification, so maybe we can remove these unless there was a particular reason for excluding?

Reply:
Thanks Andy - laziness is the answer to your query.